### ISSUE TITLE
New version doesn't recognize contact names

### ISSUE THREAD

**Issue by ubergraham** [SoFriendly/2fhey#55]
When I get a text now, the Apple/MacOS notifications show up as either the phone number or "Maybe: Person Name" and do not seem to be sync'd with my contacts list anymore.

**Comment by itsthisjustin** [comment]
I don't think this is related to 2FHey...we have nothing to do with the native notifications from iMessage
